realm/realm-library/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -564,7 +564,8 @@ public void looperThreadQuitsLooperEarly() throws InterruptedException {
 
         // Starts background looper and let it hang.
         ExecutorService executorService = Executors.newSingleThreadExecutor();
-        executorService.submit(new Runnable() {
+        //noinspection unused
+        final Future<?> future = executorService.submit(new Runnable() {
```

Comment:
`ErrorProne` reports a warning that the return value of `submit()` should not be ignored.
To suppress that, just assigning the return value to unused local variable.